NerdyNomads/Text-Savvy
# Add tests for Texts and import Test Plan

## What does this PR do?
- Added tests for the `TextBoxPopUp.js` component
- Added our Test Plan to the new `docs` folder

### Steps to test/review (optional)
1. `cd front-end`
2.  If needed, `npm install`
3. `npm test` or `npm run test`
4. All tests should pass!
## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
